src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -357,9 +358,40 @@ public static string GetDatabaseChangeVector(Transaction tx)
             return Encodings.Utf8.GetString(val.Reader.Base, val.Reader.Length);
         }
 
+        internal HashSet<string> UnusedDatabaseIds;
```

Comment:
What kind of comparer is set here? Does it have to be case insensitive one?

Reply:
it is base64, so it has to be case sensetive